app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
Can we get a list of which ones and also have an issue for this? I want to check out how they use it. Personally I don't like the idea of `code` tools going around My Documents folder where I have other stuff, than codes ...